client/components/select-dropdown/test/index.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+import { assert } from 'chai';
+import { shallow, mount } from 'enzyme';
+import sinon from 'sinon';
+
+/**
+ * Internal dependencies
+ */
+import SelectDropdown from '../index';
+
+describe( 'index', function() {
+	require( 'test/helpers/use-fake-dom' )();
+
+	describe( 'component rendering', function() {
+		it( 'should render a list with the provided options', function() {
+			const dropdownOptions = getDropdownOptions();
+			const dropdown = mount( <SelectDropdown options={ dropdownOptions } /> );
+
+			assert.equal( 'Statuses', dropdown.find( '.select-dropdown__label' ).text() );
```

Comment:
I guess that`expect` is much more popular. Some test files use `assert` from Node or `assert` from Chai. It's all fine until there is only one of them used in a single file.